rust-code-analysis-book/src/commands/nodes.md

```diff
@@ -1,52 +1,45 @@
 # Nodes
 
-**rust-code-analysis-cli** allows to extract some information from the nodes
-which compose the *Abstract Syntax Tree (AST)* of a source code.
+The rust-code-analysis-cli provides commands to analyze and extract information from the nodes in the **AST** of a source file.
 
-## Find Errors
+## Error Detection
 
-To know if there are some syntactic errors in your code, run:
+To detect syntactic errors in your code, run:
 
-```console
-rust-code-analysis-cli -p /path/to/your/file/or/directory -I "*.ext" -f -error
+```bash
+rust-code-analysis-cli -p /path/to/file/or/directory -I "*.ext" -f error
 ```
 
-The `-p` option represents the path to a file or a directory. If a directory is
-passed as input, **rust-code-analysis-cli** computes the metrics for each file
-contained in it.
-The `-I` option is a glob filter used to consider only the files written in
-the language defined by the extension of the file.
-The `-f` option instead searches all nodes of a certain type.
-In the case above, we are looking for all the erroneous nodes present in the
-code.
+- `-p`: Path to a file or directory (analyzes all files in the directory).
+- `-I`: Glob filter for selecting files by extension (e.g., `*.js`, `*.rs`).
+- `-f`: Flag to search for nodes of a specific type (e.g., errors).
 
-## Count Errors
+This command will find and list all syntax errors in the specified code.
 
-It is also possible to count the number of nodes of a certain type using the
-`--count` option:
+## Counting Nodes
 
-```console
-rust-code-analysis-cli -p /path/to/your/file/or/directory -I "*.ext" --count -error
+You can count the number of specific node types in your code by using the `--count` flag:
+
+```bash
+rust-code-analysis-cli -p /path/to/file/or/directory -I "*.ext" --count <NODE_TYPE>
 ```
+This counts how many nodes of the specified type exist in the analyzed files.
 
-## Print AST
+## Printing the AST
 
-If you want to print the AST of a source code, run the following command:
+To visualize the AST of a source file, use the `--dump` flag:
 
-```console
-rust-code-analysis-cli -p /path/to/your/file/or/directory -d
+```bash
+rust-code-analysis-cli -p /path/to/file/or/directory -I "*.ext" --dump
```

Comment:
Same here.

Also, I don't see why switching from "-d" to "--dump" here but then keep "-d" in the comment later. Also, I don't see why adding `-I "*.ext"` makes sense.